### ISSUE TITLE
check_syntax: false positive for VALUE with nested structures

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#2233]
```abap
REPORT zfoobar.

TYPES:
  BEGIN OF ty_foo,
    BEGIN OF a,
      c TYPE c,
      i TYPE i,
    END OF a,
  END OF ty_foo.

DATA(ls_foo) = VALUE ty_foo( a-c = 'X' ).
```

/zfoobar.prog.abap[11, 1]: field a-c does not exist in structure(check_syntax)

**Comment by larshp** [comment]
@mbtools try again, 2.80.8

**Comment by mbtools** [comment]
works 👍
